madeleine Description: <short summary of the patch>
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 canaima-bienvenido-gnome (4.7) stable; urgency=low
 .
   * [fb7eebf] Cambiando escudo de la img venezuela del arcerca
Author: Carlos Espinoza <carlosarmikhael@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- canaima-bienvenido-gnome-4.7.orig/gui/data/shell/01.html
+++ canaima-bienvenido-gnome-4.7/gui/data/shell/01.html
@@ -2,12 +2,12 @@
     <div style="display: block;" id="welcome">
         <div style="left: 20px; top: 30px; display: block; width: 720px; height: 549px;" id="welcome-screen" class="selected-window">	
 	      <div class="welcome-header">
-                  <h1>¡Bienvenido a Canaima 4.0!</h1>
+                  <h1>¡Bienvenido a FUNDABIT!</h1>
              </div><!-- welcome-header -->
              <div class="answer-screen" >
                    <div id="answer">
-                       <p>Canaima es un Sistema Operativo completo, versátil, novedoso, seguro, eficiente y LIBRE que cuenta con una serie de aplicaciones pensadas para facilitar tu trabajo.</p>
-                       <p>Disfruta del siguiente recorrido donde aprenderás más sobre <a href="[doc]/usr/share/canaima/help/gnome/parte_1.xml"> CANAIMA </a>  , nuestro nuevo entorno de escritorio y nuestras aplicaciones</p>
+                       <p>FUNDACION BOLIVARIANA DE INFORMATICA Y TELEMATICA.</p>
+                       <p> <a href="[doc]/usr/share/canaima/help/gnome/parte_1.xml"> MADELEINE FLORES </a> TUTOR CBIT</p>
 <table width="100%" height="110px" align="center" class="tabla">
    <tr align="center">
         <td valign="top" width="20%" class="titulo"  >
--- canaima-bienvenido-gnome-4.7.orig/gui/data/shell/02.html
+++ canaima-bienvenido-gnome-4.7/gui/data/shell/02.html
@@ -2,13 +2,13 @@
     <div style="display: block;" id="welcome">
         <div style="left: 20px; top: 30px; display: block; width: 720px; height: 549px;" id="welcome-screen" class="selected-window">	
 	      <div class="welcome-header">
-                  <h1>Aplicaciones</h1>
+                  <h1>FUNCIONES</h1>
              </div><!-- welcome-header -->
              <div class="answer-screen" >
                    <div id="answer">
                       <p>En el <a href="[doc]/usr/share/canaima/help/gnome/ayuda.xml#chap:menucanaima">Menú de Canaima</a>  puedes encontrar todas las aplicaciones por categoría instaladas en el sistema. También puedes usar el <a href="[doc]/usr/share/canaima/help/gnome/ayuda.xml#chap:Vista-de-actividades#sec:Buscador-Canaima">Buscador</a> y colocar el nombre de la aplicación para encontrarla mas rápido.</p> 
 
-<p>Si deseas instalar cualquier programa te puedes dirigir a <a href="[exec]software-center">Centro de software</a> y instalas los programas en sus versiones oficiales y libres de manera rápida y fácil</p>
+<p>Si deseas instalar cualquier programa te puedes dirigir a <a href="[exec]software-center">111</a> y instalas los programas en sus versiones oficiales y libres de manera rápida y fácil</p>
 
                           <div id="ballWrapper" style="left: 569px; top: 150px;">
 				  <div id="ball"></div>
